assets/js/base/components/cart-checkout/product-summary/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import Summary from '@woocommerce/base-components/summary';
+import { getSetting } from '@woocommerce/settings';
+
+/**
+ * Returns an element containing a summary of the product.
+ */
+const ProductSummary = ( { shortDescription = '', fullDescription = '' } ) => {
+	const source = shortDescription ? shortDescription : fullDescription;
+
+	if ( ! source ) {
+		return null;
+	}
+
+	const countType = getSetting( 'wordCountType', 'words' );
+
+	return (
+		<Summary source={ source } maxLength={ 15 } countType={ countType } />
+	);
+};
```

Comment:
maybe avoid variable declaration here and just invoke `getSetting` directly for the `countType` prop?

```suggestion
	return (
		<Summary source={ source } maxLength={ 15 } countType={ getSetting( 'wordCountType', 'words' ) } />
	);
};
```